_utils/path_firefox.ini

```diff
@@ -0,0 +1,17 @@
+[darwin]
+PATH_FIREFOX_GR="/Applications/Firefox/Contents/MacOS"
+PATH_FIREFOX_NN="/Applications/FirefoxNightly.app/Contents/MacOS"
+PATH_FIREFOX_DE="/Applications/FirefoxDeveloperEdition/Contents/MacOS"
```

Comment:
that was my own sloppy oversight with the file i gave you. yea, i think you're right.  let's go with firefox and firefox.exe.
here's an old bug explaining the difference, but it sounds like firefox-bin is legacy:
https://bugzilla.mozilla.org/show_bug.cgi?id=658850#c9